### PR TITLE
Update Ruby to 2.2.3

### PR DESCRIPTION
**Pull request by alexstophel**
Yay! A new version of Ruby was released a couple of days ago. It's
important that new projects start with the latest version, so we should
use 2.2.3 for Suspenders.

* Updated .ruby-version file to Ruby 2.2.3.
* Updated .travis.yml file to Ruby 2.2.3.

### CODE REVIEW

**Comment by tute** [comment]
Thanks! Can you add a note of the change to NEWS.md too please? Thank you!

**Comment by alexstophel** [comment]
:+1:

Just updated the commit, @tute.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+1.30.1 (August 20, 2015)
```
Comment (tute) [inline NEWS.md]: Let's correct this to `1.31.0 (unreleased)`, and I'll merge. Thank you very uch!

Comment (alexstophel) [inline NEWS.md]: :+1:

**Comment by tute** [comment]
Thank you! :)

**Pull request merged**